library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -23,6 +23,14 @@
         @include display-flex;
         @include justify-content(space-between);
 
+        .linkStyle{
+            &:hover, &:link, &:visited{
+                text-decoration: none !important;
+                color: black !important;
```

Comment:
Are you trying to reset this colour to the default text colour? If so, please use `$text-color`.